photutils/psf/models.py

```diff
@@ -497,8 +497,12 @@ def evaluate(self, x, y, flux, x_0, y_0, use_oversampling=True):
 
 class EPSFModel(FittableImageModel):
     """
-    A subclass of `FittableImageModel`. A fittable ePSF model.
+    A class that models an effective PSF (ePSF). This is a subclass of
```

Comment:
Nit:  Split the second sentence into a new paragraph so there's a nice one-line summary to start the doctring.  (Unfortunately multi-line suggestions are not available in github yet).